go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -289,13 +290,84 @@ func (m *MasterController) handleOverlayPort(node *kapi.Node, annotator kube.Ann
 		}
 	}
 
+	// create a reroute on the ovn_cluster_router
+	err = createLogicalRouterPolicyForNode(node, subnets)
+	if err != nil {
+		return fmt.Errorf("failed to create logical router policy: %v", err)
+	}
+
+	return nil
+}
+
+func createLogicalRouterPolicyForNode(node *kapi.Node, subnets []*net.IPNet) error {
+	// get all the windows subnets as the dest for lr-policy match
+	var dest string
+	var nextHop string
+	var src string
+	for _, hybridSubnet := range config.HybridOverlay.ClusterSubnets {
+		if !utilnet.IsIPv6CIDR(hybridSubnet.CIDR) {
+			dest = dest + fmt.Sprintf("|| ip4.dst == %s", hybridSubnet.CIDR.String())
```

Comment:
Instead of doing the [3:] below, I'd rather have this block check `if len(dest) > 0 { dest = dest + " || " }` before adding the ip4.dst... stuff

Reply:
done